src/dashboard/infrastructure/integrations/site-kit.php

```diff
@@ -176,6 +176,45 @@ private function can_read_data( array $module ): bool {
 		return $module['can_view'] || $this->is_owner( $module['owner'] );
 	}
 
+	/**
+	 * Gets the prefix for the client side cache key.
+	 *
+	 * Cache key is scoped to user session and blog_id to isolate the
+	 * cache between users and sites (in multisite).
+	 *
+	 * @return string
+	 */
+	private function get_storage_prefix() {
+		$current_user  = \wp_get_current_user();
+		$auth_cookie   = \wp_parse_auth_cookie();
+		$blog_id       = \get_current_blog_id();
+		$session_token = isset( $auth_cookie['token'] ) ? $auth_cookie['token'] : '';
+
+		return \wp_hash( $current_user->user_login . '|' . $session_token . '|' . $blog_id );
+	}
+
+	/**
+	 * Gets the Time To Live for each widget's cache.
+	 *
+	 * @return array<string, array<string, int>> The cache TTL for each widget.
+	 */
+	private function get_widgets_cache_ttl() {
```

Comment:
I really doubt having this in the backend is worth the tying it to the widgets types part.
But I suppose the future will tell.